### ISSUE TITLE
Integrate new blame with Tracker

### ISSUE THREAD

**Issue by TPolzer** [uv48uson/amos-ss15-proj3#94]
In a48b2d4566b56ba33b529ad31f52218afe332d91 I have implemented the necessary logic to connect the new blame with Tracker. It turns out however, that the tests for Tracker are tied to the current implementation in ways that I do not understand (some mock objects clash with real objects...).

@yadimon could you fix this, i.e. just call the new wordBlame from Tracker and adapt the tests?

**Comment by TPolzer** [comment]
If somebody knows how to modify the Tracker tests, this is needed for integration in Tracker:
``` java
public List<AnnotatedLine> getBlame(String path) throws IOException, GitAPIException {
        GitBlameOperation op = new GitBlameOperation(vcsClient, path, (s, i) -> null);
        return AnnotatedLine.wordsToLine(op.wordBlame(), dataSource);
    }
```

**Comment by GnuTalux** [comment]
nice to hear, that I am not the only one having troubles modifying TrackerTest ^^

**Comment by yadimon** [comment]
@TPolzer  i ll try to fix today.

**Comment by GnuTalux** [comment]
If I am finished with my implementation (which I want to put in Tracker),
I will test it. 
As I look over the TrackerTest, I had no idea, how I can test my code in future.
:)) 

but, as I see, I will commit (currently not finsihed) implementation to next sprint.

@yadimon thank for wanting to help me

**Comment by yadimon** [comment]
@TPolzer did you change the Test repo from resources?
pls dont change it, many tests are based on it. So create new one (even if its similar one), or better, create bigger one.

**Comment by TPolzer** [comment]
I changed it after confirming that all tests still pass.

**Comment by TPolzer** [comment]
Feel free to put it in an extra repository if something breaks for you.

**Comment by yadimon** [comment]
merged by e03236e2305b0e58254ba0f4374e113046d70d15
but i think we have to write for the blame and the matrix much more good tests.

**Comment by TPolzer** [comment]
Thank you.
